ci/L0_multi_gpu/test.sh

```diff
@@ -42,7 +42,7 @@ rm -rf models && mkdir -p models
 cp -r ${SAMPLE_MODELS_REPO}/vllm_model models/vllm_opt
 sed -i '3s/^/    "tensor_parallel_size": 2,\n/' models/vllm_opt/1/model.json
 
-pip3 install tritonclient grpcio nvidia-ml-py3
+python3 -m pip install --upgrade pip && pip3 install tritonclient[grpc] nvidia-ml-py3
```

Comment:
One more quick comment: for this and the other pip upgrade... is this necessary? It looks like the QA container should have [the latest pip](https://github.com/triton-inference-server/server/blob/d33eaa5a153f5333374046f12826aac5ffd5ed52/Dockerfile.QA#L62) already. Is this something that needs to be added to the vllm backend container instead?

Reply:
@dyastremsky  I would advise not to include the additional libraries in our vLLM backend container that we release on NGC as it is expected to be a runtime image similar to our full-fat container image.
For simplicity, we didn't create a separate QA container as we do in our general container CI. 
We might have done that if there were significant dependencies and setup that were needed to be conducted.

Reply:
I would suggest let's start from here and see how it goes.. We may consider using a common requirements.txt file to install packages in several tests.. Or go with a separate QA container build in our pipeline.

Reply:
Okay, sounds good! This just seems like the wrong place to have the responsibility for upgrading pip. We can do it now for a less-scalable option and revisit creating a vLLM-QA container, if needed later.